README.md

```diff
@@ -1,50 +1,199 @@
-# Armbian #
+<h3 align=center><a href="#armbian-build-engine"><img src=".github/armbian-logo.png" alt="Armbian logo" width="144"></a><br>
+build engine</h3>
 
-Debian based Linux for ARM based single-board computers
-  
-[https://www.armbian.com](https://www.armbian.com "Armbian")
+<p align=right>&nbsp;</p>
+
+## Table of contents
+
+- [What this project does?](#what-this-project-does)
+- [What do you need to get started?](#what-do-you-need-to-get-started)
+- [How to build an image or a kernel?](#how-to-build-an-image-or-a-kernel)
+- [Compare with industry standards](#compare-with-industry-standards)
+- [Where to download prebuilt images?](#where-to-download-prebuilt-images)
+- [Additional information](#additional-information)
+- [Build engine overview](#build-engine-overview)
+- [Support](#support)
+- [Contribute](#contribute)
+- [Social](#social)
+- [Credits](#credits)
+- [Sponsors](#sponsors)
+
+<p align=right>&nbsp;</p>
+
+## What this project does?
+Help with maintaining
```

Comment:
Another copy & paste artifact?

Apart from that I think the title "What does this project do" sounds nicer than "What this project does" but that is nitpicking and something a native english speaker should look at.